### DISCUSSION TITLE
fragment size

### DISCUSSION THREAD

**Discussion by e-coral** [Ruitulyu/KAS-Analyzer#21] [Q&A] [answered]
Hi,
Please can you explain why the fragment size in the epic2 commands in map_peakscalling.sh is 300? I was expecting it to be 150. 
Thanks
-Emmon

**Comment by Ruitulyu** [comment, marked as answer]
Hi Emmon,
I think the fragment size should be an estimated number, which should not
make a big difference for the peak calling. You can also change the
fragment size from 300bp to 150bp and check the difference.
Best.
Ruitu

On Wed, Jan 10, 2024 at 8:45 AM E Croot ***@***.***> wrote:

> Hi,
> Please can you explain why the fragment size in the epic2 commands in
> map_peakscalling.sh is 300? I was expecting it to be 150.
> Thanks
> -Emmon
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Ruitulyu/KAS-Analyzer/discussions/21>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APDPVGHPBLCVGSQPAQO2IGLYN2SR7AVCNFSM6AAAAABBU5HJVSVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGA2TKNZVG4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Nutricula, based in Chicago.

**Reply by e-coral** [reply]
Thanks
